guides/common/modules/proc_updating-and-refreshing-red-hat-subscription-manifests.adoc

```diff
@@ -16,3 +16,9 @@ Alternatively, you can import an updated manifest that contains the changes.
 . In the {ProjectWebUI}, navigate to *Content* > *Subscriptions*.
 . In the Subscriptions window, click *Manage Manifest*.
 . In the Manage Manifest window, click *Refresh*.
+
+[NOTE]
+====
+The {Project} web UI notifies you before your subscription manifest expires. 
```

Comment:
Typically, when there is a notification in the web UI, that should be enough. If the notification isn't clear, the fix should not be to add a note to docs but rather to reword the web UI notification to make it clearer.